modules/ROOT/pages/testing-oauth-modules.adoc

```diff
@@ -0,0 +1,71 @@
+= Testing OAuth Enabled Connectors
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule runtime engine supports authorization through various OAuth grant types including `Authorization Code`. This grant type allows an external system, for example your Mule application, to operate on behalf of a user without the need to share the user's credentials. However, because the `Authorization Code` grant type is designed to force human intervention in the authentication process that identifies users using OAuth (OAuth dance), building an automated test for an application using this grant type is challenging.
+
+You can use the Storage Event processor to bypass the whole OAuth dance phase by manually providing a valid access token. It is your responsibility to manually obtain (and eventually update) such token.
+
+Consider the following configuration of the Salesforce connector using OAuth authentication and a user defined object store to hold the tokens:
```

Comment:
```suggestion
Consider the following configuration of the Anypoint Connector for Salesforce (Salesforce Connector) using OAuth authentication and a user-defined object store to hold the tokens:
```

Reply:
For further connector reference guidelines, see the Mulesoft Technical Documentation Style Reference:

connector names
In all new or revised text, use the following official naming standard for Anypoint Connectors:

     Full product name: Anypoint Connector for XYZ
     Short product name: XYZ Connector

Do not precede either the full name or the short name with an article (a, an, the).

You can refer to a connector in general by using “a connector.” You can refer to a specific connector for which you have already established context as “the connector” or “this connector."

Reply:
Also, I wonder if this sentence really connects the need to obtain a token and then how to use it?
The following example shows a "user-defined object store to hold the tokens," but we have not told the audience that they should create a store to hold the token (tokens?) that they obtain, monitor?, and update as they expire.
Can we assume that the audience knows about creating the store?
And does the example show more than one token? I can't identify what syntax represents the token or tokens, but that is probably because I am as yet unfamiliar with this type of syntax....